src/middlewared/middlewared/plugins/enclosure_/map.py

```diff
@@ -96,7 +96,7 @@
             MappingSlot(0, 15, False),
         ]),
     ]),
-    ProductMapping(re.compile(r"TRUENAS-R20$"), [
+    ProductMapping(re.compile(r"TRUENAS-R20B?$"), [
```

Comment:
Should this not be `TRUENAS-R20?$`?

Reply:
No R20B? means zero or more B? so Matches R20 and R20B?

Reply:
Hmm to many question marks

Reply:
I see now that `R20B` is the same as `R20`. Can we add a simple comment inside this mapping saying `R20` and `R20B` use same chassis, please?

Reply:
Good?

Reply:
perfect